templates/footer.html

```diff
@@ -1,9 +1,15 @@
   <footer>
       <div class="wrapper">
+        <div class="social-links">
+          <a target="_blank" href="https://github.com/Polymer" rel="Github"><img src="/images/logos/logo_github.svg" /></a>
+          <a target="_blank" href="https://twitter.com/polymer" rel="Twitter"><img src="/images/logos/logo_twitter.svg" /></a>
+          <a target="_blank" href="https://groups.google.com//forum/#!forum/polymer-dev" rel="Email"><img src="/images/logos/logo_email.svg" /></a>
+          <a target="_blank" href="https://polymer-slack.herokuapp.com" rel="Slack"><img src="/images/logos/logo_slack.svg" /></a>
```

Comment:
I believe `rel` should be `noopener` (or one of the [link types](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types)) and use `aria-label` for the label, e.g.
```html
<a target="_blank" href="https://polymer-slack.herokuapp.com" rel="noopener" aria-label="Slack"></a>
```

Reply:
Thanks for the tips, i've pushed changes for rel & aria-label.